hack/prowimagebuilder/main.go

```diff
@@ -150,7 +150,11 @@ func allBaseTags() ([]string, error) {
 	if err != nil {
 		return nil, err
 	}
-	return append(defaultTags, gitTag), nil
+	// Add a `ko-<GIT_TAG>` tag so that it's easy to identify images built from
+	// ko vs. images built from bazel, in case there is a revert needed.
+	// TODO(chaodaiG): remove `ko-` tag once the images produced by ko proved to
+	// be working
+	return append(defaultTags, gitTag, "ko-"+gitTag), nil
```

Comment:
Will this cause problems with the autobumper? I think we parse this tag in a few places.

Reply:
The image used in prod will still be `YYYYMMDD-<GIT_HASH>`, not the tag with `ko-` prefix. Adding this tag just for us to be able to easily tell which image is produced by whom from UI

Reply:
Gotcha :+1: